models/pod.js

```diff
@@ -72,5 +73,36 @@ export default {
         }
       }, { root: true });
     };
+  },
+
+  containerStateDisplay() {
+    return (container) => {
+      const state = Object.keys(container.state || {})[0];
+
+      if ( REMAP_STATE[state] ) {
+        return REMAP_STATE[state];
+      }
+
+      return state;
+    };
+  },
+
+  containerStateColor() {
+    return (container) => {
+      const state = this.containerStateDisplay(container);
+      const key = (state || '').toLowerCase();
+      let color;
+
+      if ( STATES[key] && STATES[key].color ) {
+        color = this.maybeFn(STATES[key].color);
+      }
+
+      if ( !color ) {
+        color = DEFAULT_COLOR;
+      }
+
+      return `text-${ color }`;
```

Comment:
You could pull this (and stateDisplay) out into a standalone function that's exported by resource-instance and then import/call it here instead of duplicating all/most of the code and exporting its constants..